product_docs/docs/efm/4/04_configuring_efm/05_using_vip_addresses.mdx

```diff
@@ -72,75 +72,79 @@ ping.server.command=/bin/ping -q -c3 -w5
-
-```text
-# /bin/ping -q -c3 -w5 172.24.38.239
-PING 172.24.38.239 (172.24.38.239) 56(84) bytes of data.
---- 172.24.38.239 ping statistics ---
-3 packets transmitted, 3 received, 0% packet loss, time 1999ms
-rtt min/avg/max/mdev = 0.023/0.025/0.029/0.006 ms
-```
-
-No packet loss occurs.
-
-4.  Use the `efm_address del` command to release the address on the primary node and confirm the node was released with ip address:
-
-```text
-# efm_address del eth0 172.24.38.239/24
-# ip address
-eth0 Link encap:Ethernet HWaddr 22:00:0A:89:02:8E
-inet addr:10.137.2.142 Bcast:10.137.2.191
-...
-```
-
-The output from this step doesn't show an eth0 interface.
-
-5.  Repeat step 3, this time verifying that the standby and witness don't see the VIP in use:
-
-```text
-# /bin/ping -q -c3 -w5 172.24.38.239
-PING 172.24.38.239 (172.24.38.239) 56(84) bytes of data.
---- 172.24.38.239 ping statistics ---
-4 packets transmitted, 0 received, +3 errors, 100% packet loss,
- time 3000ms
-```
-
-100% packet loss occurs. Repeat this step on all nodes.
-
-6.  Repeat step 2 on all standby nodes to assign the VIP to every node. You can ping the VIP from any node to verify that it's in use.
-
-```text
-# efm_address add4 eth0 172.24.38.239/24
-# ip address
-<output truncated>
-eth0 Link encap:Ethernet HWaddr 36:AA:A4:F4:1C:40
-inet addr:172.24.38.239 Bcast:172.24.38.255
-...
-```
+1. Ping the VIP from all nodes to confirm that the address isn't already in use:
+
+   ```text
+   # /bin/ping -q -c3 -w5 172.24.38.239
+   PING 172.24.38.239 (172.24.38.239) 56(84) bytes of data.
+   --- 172.24.38.239 ping statistics ---
+   4 packets transmitted, 0 received, +3 errors, 100% packet loss,
+   time 3000ms
+   ```
+
+   You see 100% packet loss. 
+   !!!important
+   Failover Manager uses the exit code of the command to determine whether or not the address was reachable. In this case the exit code isn't zero. If using a different command, it must return a non-zero exit code if the address isn't reachable.
+
+2. Run the `efm_address add4` command on the Primary node to assign the VIP, and then confirm with ip address:
+
+   ```text
+   # efm_address add4 eth0 172.24.38.239/24
+   # ip address
+   <output truncated>
+   eth0 Link encap:Ethernet HWaddr 36:AA:A4:F4:1C:40
+   inet addr:172.24.38.239 Bcast:172.24.38.255
+   ...
+   ```
+
+3. Ping the VIP from the other nodes to verify that they can reach the VIP:
+
+   ```text
+   # /bin/ping -q -c3 -w5 172.24.38.239
+   PING 172.24.38.239 (172.24.38.239) 56(84) bytes of data.
+   --- 172.24.38.239 ping statistics ---
+   3 packets transmitted, 3 received, 0% packet loss, time 1999ms
+   rtt min/avg/max/mdev = 0.023/0.025/0.029/0.006 ms
+   ```
+
+   No packet loss occurs. 
+   !!!Important
+   Failover Manager uses the exit code of the command to determine whether or not the address was reachable. In this case the exit code is zero. If using a different command, it must return a zero exit code if the address is reachable.
```

Comment:
@EFM-Bobby , if you are ok with these edits I'll make them to the other note too.
```suggestion
   Failover Manager uses the exit code of the ping command to determine whether or not the address was reachable. In this case the exit code is zero. If using a command other than ping, it must return a zero exit code if the address is reachable.
```

Reply:
Yes, that looks good to me. Thank you.